src/deriver/tom/embeddings.py

```
@@ -1,56 +1,554 @@
+                            'last_accessed': doc.h_metadata.get("last_accessed"),
+                            'access_count': doc.h_metadata.get("access_count", 0),
+                            'accessed_sessions': doc.h_metadata.get("accessed_sessions", [])
+                        }
+                        context[level].append(fact_info)
+                        seen_facts.add(normalized_content)
+                    else:
+                        logger.debug(f"Skipping duplicate fact during retrieval: {doc.content[:50]}...")
+            
+            return context
+            
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)